src/main/webapp/BusinessForm.html

```diff
@@ -1,68 +1,71 @@
 <!DOCTYPE html>
 <html>
-  <head>
+
+<head>
     <meta charset="UTF-8">
     <title>Webapp Project</title>
     <link rel="stylesheet" href="BusinessForm.css">
     <script src="BusinessForm.js"></script>
 
-    
-  </head>
-  <header>
-      <h1 style="font-size:22px;">Landing Page Creation: Business Form</h1>
-  </header>
-  <body>
-    <aside class = "aside">
-        <section>
-            <h2 style="font-size:22px;">Guidelines</h2>            
-                    <form action="/form-handler" id="guidelines">
-                    <p>Enter guidelines here:</p>
-                    <textarea name="text-input" rows="20" cols="40" placeholder="We follow..."></textarea>
-                    <br/>
-                </form>
-        </section>
-        <section>
-            <h2 style="font-size:22px;">Business Description</h2>            
-                    <form action="/form-handler" id="description">
-                    <p>Enter business description here:</p>
-                    <textarea name="text-input" rows="20" cols="40" placeholder="Our work entails..."></textarea>
-                    <br/>
-                </form>   
-        </section>
-    </aside>
-    <main>
-        <article>
+
+</head>
+<header>
+    <h1 style="font-size:22px;">Landing Page Creation: Business Form</h1>
+</header>
+
+<body>
+    <form>
```

Comment:
Form tag needs to have method and action properties. <form action="/form-handler" method="POST">